### PR TITLE
ci: fix rook cluster version fetching

### PR DESCRIPTION
**Pull request by humblec**
As part of https://github.com/ceph/ceph-csi/pull/1237/ there was
a patching enabled for the ceph cluster deployed, however due to
an error in the version fetching logic, the patching was not applied

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: scripts/rook.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 
 rook_version() {
-	echo "${ROOK_VERSION#?v}" | cut -d'.' -f"${1}"
+	echo "${ROOK_VERSION#?}" | cut -d'.' -f"${1}"
```
Comment (nixpanic) [inline scripts/rook.sh:10]: why leave the `#$` in here?

Comment (humblec) [reply scripts/rook.sh:10]: @nixpanic in case of first iteration on the string `v14.2.10` it was giving back `v1`.

Comment (nixpanic) [reply scripts/rook.sh:10]: Ah, yes. But in this case you are cutting the 1st character from the ROOK_VERSION, which may not be the `v` for all (future) release tags. It is a little safer to use `echo "${ROOK_VERSION#v}"` that will only drop an initial `v`.

Comment (humblec) [reply scripts/rook.sh]: Sure. done.. ptal

**Comment by humblec** [comment]
@nixpanic @Madhu-1 ptal

**Pull request merged**